### PR TITLE
Replacing MSDN links with docs relative

### PR DESCRIPTION
**Pull request by nemrism**
Contributes to #2037

### CODE REVIEW

File: docs/framework/app-domains/assembly-placement.md [hunk 1]
```diff
@@ -18,5 +18,5 @@ For most .NET Framework applications, you locate assemblies that make up an appl
 ## See Also  
  [How the Runtime Locates Assemblies](../../../docs/framework/deployment/how-the-runtime-locates-assemblies.md)  
  [Configuring Apps](../../../docs/framework/configure-apps/index.md)  
- [Advanced COM Interoperability](https://msdn.microsoft.com/library/3ada36e5-2390-4d70-b490-6ad8de92f2fb)  
+ [Advanced COM Interoperability](../../../docs/framework/interop/index.md)
```
Comment (rpetrusha) [inline docs/framework/app-domains/assembly-placement.md]: The title should be changed to "Interoperating with unmanaged code"

File: docs/framework/common-client-technologies/how-to-configure-client-application-services.md [hunk 2]
```diff
@@ -8,7 +8,7 @@ ms.assetid: 34a8688a-a32c-40d3-94be-c8e610c6a4e8
...
   
- You can enable and configure client application services on the **Services** page of the **Project Designer**. The **Services** page updates values in your project's App.config file. To access the **Project Designer**, use the **Properties** command on the **Project** menu. For more information about the **Services** page, see [Services Page, Project Designer](https://msdn.microsoft.com/library/bb398109).  
+ You can enable and configure client application services on the **Services** page of the **Project Designer**. The **Services** page updates values in your project's App.config file. To access the **Project Designer**, use the **Properties** command on the **Project** menu. For more information about the **Services** page, see [Services Page, Project Designer](/visualstudio/ide/reference/signing-page-project-designer).
```
Comment (rpetrusha) [inline docs/framework/common-client-technologies/how-to-configure-client-application-services.md]: signing-page --> services-page

File: docs/framework/common-client-technologies/how-to-configure-client-application-services.md [hunk 3]
```diff
@@ -137,7 +137,7 @@ This topic describes how to use the Visual Studio **Project Designer** to enable
 ## See Also  
  [Client Application Services](../../../docs/framework/common-client-technologies/client-application-services.md)  
  [Client Application Services Overview](../../../docs/framework/common-client-technologies/client-application-services-overview.md)  
- [Services Page, Project Designer](https://msdn.microsoft.com/library/bb398109)  
+ [Services Page, Project Designer](/visualstudio/ide/reference/signing-page-project-designer)
```
Comment (rpetrusha) [inline docs/framework/common-client-technologies/how-to-configure-client-application-services.md]: signing --> services

**Review by rpetrusha — CHANGES_REQUESTED** [review]
Thanks, @nemrism, for continuing to submit PRs that replace MSDN links with links to docs.microsoft.com. I've left a couple of comments and suggestions.

File: docs/framework/common-client-technologies/how-to-configure-client-application-services.md (hunk 3 above)
Comment (nemrism) [reply docs/framework/common-client-technologies/how-to-configure-client-application-services.md]: we should check if I made the same mistake when replacing links pointing to the "Signing Page, Project Designer"

Comment (nemrism) [reply docs/framework/common-client-technologies/how-to-configure-client-application-services.md]: I didn't spot others, but I'll let you double check @rpetrusha

**Review by rpetrusha — APPROVED** [review]

**Comment by rpetrusha** [comment]
Thanks, @nemrism, for making the additional changes. I apologize that it's taken me so long to get back to your PR. In any case, I'll merge now, and your changes should be live on docs.microsoft.com in the next day or two.

**Pull request merged**